README.md

```diff
@@ -10,38 +10,40 @@ Experiment tracking, model registry, data versioning, and live model monitoring
 
 ## What will be logged to Neptune?
 
-* Hyper-parameters
-* Losses & metrics
-* Training code (Python scripts or Jupyter notebooks) and git information
+* Hyperparameters
+* Losses and metrics
+* Training code (Python scripts or Jupyter notebooks) and Git information
 * Dataset version
-* Model Configuration, architecture, and weights
-* [other metadata](https://docs.neptune.ai/you-should-know/what-can-you-log-and-display)
+* Model configuration, architecture, and weights
+* [Other metadata](https://docs.neptune.ai/you-should-know/what-can-you-log-and-display)
```

Comment:
Should we use this link https://docs.neptune.ai/logging/what_you_can_log/ ?

Reply:
Yes, missed that one.